pom.xml

```diff
@@ -250,6 +327,7 @@
     <module>grpc-google-cloud-bigquerystorage-v1</module>
     <module>google-cloud-bigquerystorage</module>
     <module>google-cloud-bigquerystorage-bom</module>
+    <module>samples</module>
```

Comment:
We don't want this in the main `<modules>` list -- we don't want others to have to run the samples tests to contribute to the main artifacts. Instead, we'll put it in the `enable-samples` profile like: https://github.com/googleapis/java-logging/blob/6f4fae2141662f0421364ea25f738366f3feaf41/pom.xml#L312-L319